examples/basic.html

```diff
@@ -27,6 +34,22 @@
         This is required.
       </p>
     </section>
+    <section class="introductory" id="overview">
+      <h2>Overview</h2>
+      <p>
+        This section's class means it will not appear in the
+        <abbr title = "Table of Contents">TOC</abbr>. Example Link to Section
+        <a href=#logo></a>.
+      </p>
+    </section>
+    <section id="tof"></section>
```

Comment:
We probably want the `tof` at the end of the document, as it's more common to find it there.